.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,39 @@
+name: Bug
+description: Suggest an issue or typo either in the specification, or on the website
+body:
+  - type: dropdown
+    id: area
+    attributes:
+      label: Problem area
+      description: What is this an issue with?
+      multiple: true
+      options:
+        - Specification
+        - designtokens.org
+        - GitHub repo
+        - Other
+    validations:
+      required: true
+  - type: textarea
+    id: summary
+    attributes:
+      label: Problem
+      description: Describe the issue in detail.
+    validations:
+      required: true
+  - type: textarea
+    id: expected
+    attributes:
+      label: Suggested fix
+      description: What’s the suggested fix here?
+    validations:
+      required: true
```

Comment:
maybe suggested fix isn't required? since someone might have a bug without a solution in mind.